### DISCUSSION TITLE
Adding to an existing Security Group

### DISCUSSION THREAD

**Discussion by DrJohnDale** [nrdg/cloudknot#303] [Q&A]
Hi,

Is there any way, when creating a new cloudknot, to set the security group to use?

We want to use a batch job to access am aws database instance where access is controlled via a firewall.

Thanks,

John

**Comment by richford** [comment]
Hi @DrJohnDale, thanks for bringing this up. This isn't currently possible, but could be added. I'll convert this to an issue.

**Reply by DrJohnDale** [reply]
Thanks
